nx-X11/programs/Xserver/hw/nxagent/Screen.c

```diff
@@ -1910,6 +1913,9 @@ N/A
                            nxagentIconPixmap,
                            argv, argc, sizeHints);
 
```

Comment:
Above the below code block, I read:

```
    /* FIXME: deprecated, replaced by XSetWmProperties() */
    XSetStandardProperties(nxagentDisplay,
                           nxagentDefaultWindows[pScreen->myNum],
                           nxagentWindowName,
                           nxagentWindowName,
                           nxagentIconPixmap,
                           argv, argc, sizeHints);
```

Have you tried replacing XSetStandardProperties() call by X(utf8)SetWmProperties()? Mayb the below code calling XSetWMName() would be obsolete then.

Can you check that?

Reply:
I apologize for not having responded.
Now I'm a little busy. As soon as I'm free,
I will try to test this function.

P.S. It seems I tried to use this function,
but it did not work. Unfortunately, I can not remember.

Reply:
I checked the "Xutf8SetStandardProperties" function, it really works.
But with 'XSetStandardProperties ()' nxagent is terminated.

I updated the commit.